exdb/views.py

```diff
@@ -51,9 +51,11 @@ def get_hs_queryset(self):
         experience_approvals = ExperienceApproval.objects.filter(
             approver=self.request.user, experience__status='ad'
         )
+        next_approver_queue = Q(next_approver=self.request.user) & ~Q(status="dr")
```

Comment:
Thinking about it, this should probably be:
`Q(next_approver=self.request.user) & Q(status='pe')`
Thoughts?